setup.cfg

```diff
@@ -0,0 +1,56 @@
+[metadata]
+# would we ever want to use pyproject.toml ?
```

Comment:
I want to be a cool kid, but I really don't know that much about it.

> it's the future of packaging in the Python ecosystem

but 

> This isn't the first time I have seen setuptools users use pyproject.toml because they were "told to by `<`insert name of tool`>`" without knowing the entire point behind the file.

https://snarky.ca/what-the-heck-is-pyproject-toml/